tests/ui/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [tool:pytest]
-addopts = -r=a -vs --showlocals --tb=short
+addopts = -r=a -vs --showlocals --tb=short --html=ui-test.html --self-contained-html
 sensitive_url = mozilla\.(com|org)
 xfail_strict = true
 DJANGO_SETTINGS_MODULE = settings
-base_url = http://olympia.test:80
+base_url = http://olympia.local.test:4000
```

Comment:
Imho this should be `olympia.test` only, we don't document `olympia.local.test` anywhere.

Reply:
I think if I did  this it would use the other alias for nginx that is set within the docker-compose.

Reply:
Not sure I follow, what do you mean? I don't grasp the full picture regarding the hostnames, can you share a few more details?